builder/make.sh

```diff
@@ -7,3 +7,26 @@
 
 # Start parallel build
 make -j $(nproc) V=sc world
+
+IMG=$(find /home/build/openwrt/bin -name nextsecurity\*-x86-64-generic-ext4-combined-efi.img.gz);
```

Comment:
Can we loop on all "nextsecurity\*-generic-ext4-combined-efi.img.gz" images?
For now we only have x86-64 target, but in the future...who knows!